src/test/java/com/tratif/pdfgen/document/DocumentBuilderTest.java

```diff
@@ -0,0 +1,52 @@
+package com.tratif.pdfgen.document;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class DocumentBuilderTest {
+
+    @Test
+    public void isProperPdfFile() {
+        byte[] bytes = Document.fromStaticHtml("<h1>hello, world!</h1>")
+                .toPdf();
+
+        assertThat(bytes).isNotNull();
+        assertThat(bytes.length).isGreaterThan(4);
+
+        // header
```

Comment:
(I'm assuming that the code below will be refactored into PdfAssert --  in a separate pull request)